package.json

```diff
@@ -1,79 +1,44 @@
 {
   "name": "@kodingdotninja/chakra-icons",
   "version": "0.0.0",
-  "workspaces": [
```

Comment:
This is will be impacted with multi-semantic-release. But I want to find another solutions for handle release.

Reply:
We could restore the workspaces field